Source/Server/MqttService.h

```diff
@@ -25,6 +26,7 @@ namespace MQTT {
 
 
 			MqttClient* GetClientState(const std::string& clientId);
+			std::string GenerateUniqueId();
```

Comment:
Remove from header file

Reply:
Resolved by deleting method. Not used anymore